src/auto/repo_finder_result.rs

```diff
@@ -3,6 +3,7 @@
 // DO NOT EDIT
 
 use std::cmp;
+use glib::translate::*;
```

Comment:
I tried to find out what's wrong about this in `gir`, but failed so far.

I think we manually added this in the past too, even though the git-log does not mention that.

Reply:
Yeah sorry I think I did do that...